src/context/SelectedBooking.js

```diff
@@ -64,9 +67,14 @@ class SelectedBookingProvider extends React.Component<Props, State> {
       onSelectBooking: this.onClickSelect,
       closeAllBooking: this.closeAllBooking,
       onDisplayAll: this.onClickAllBooking,
+      isChatEnabled: this.isChatEnabled,
     };
   }
 
+  isChatEnabled = () => {
+    return this.props.enableChat || false;
```

Comment:
if `enableChat` is required prop, why fallback option? `this.props.enableChat || false`